ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -346,6 +347,8 @@ measurement_descriptor:
   superheat: "The amount a gas is heated above its boiling point."
   thermal: "Relating to the transfer of heat."
   tilt: "Degree of radial rotation."
+  wavespeed: "Indicates the number of frequency sweeps per minute and it varies with frequency bands."
```

Comment:
this and the following term are no longer used in the PR. please remove.

Reply:
Yes Removed